pkgs/by-name/ne/neomutt/package.nix

```diff
@@ -70,17 +72,19 @@ stdenv.mkDerivation (finalAttrs: {
     ++ lib.optional withNotmuch notmuch;
 
   nativeBuildInputs = [
-    docbook_xsl
     docbook_xml_dtd_42
+    docbook_xsl
     gettext
+    libkrb5
     libxml2
     libxslt.bin
     makeWrapper
+    perl
+    pkg-config
     tcl
+    w3m
     which
     zlib
-    w3m
-    pkg-config
```

Comment:
I just sorted the build list here, the only actual additions were perl and libkrb5